### PR TITLE
[WIP] Hearing Worksheet | Edited flag on issue edits

### PR DESCRIPTION
**Pull request by nolawi**
Connects #3073 

screen shot 2017-09-21 at 2 51 13 pm

### CODE REVIEW

File: client/app/hearings/reducers/index.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const newHearingIssueState = (state, action, spec) => {
     worksheet: {
       streams: {
         [action.payload.appealId]: {
+          edited: { $set: true },
```
Comment (joofsh) [inline client/app/hearings/reducers/index.js]: Is this field already added to the `initialState` and set to false?

Comment (nolawi) [reply client/app/hearings/reducers/index.js]: There is no need for that - it doesn't have to exist till there is a change. The auto save will change it to false. I think based on my understanding

Comment (joofsh) [reply client/app/hearings/reducers/index.js]: You're correct it doesn't need to exist, but I would prefer if it did (if it's not insanely complex to do). It's helpful for devs to see the whole state from the very beginning. It helps give context on how the state can & will be updated

Comment (nolawi) [reply client/app/hearings/reducers/index.js]: Got it! Will fix

**Comment by nolawi** [comment]
closing this for now.. we need to do some refactor before its ready for dev

**Pull request closed without merging**